doc/lua_api.md

```diff
@@ -6579,8 +6581,10 @@ Environment access
       Difference between `"A*"` and `"A*_noprefetch"` is that
       `"A*"` will pre-calculate the cost-data, the other will calculate it
       on-the-fly
-* `core.spawn_tree (pos, {treedef})`
+* `core.spawn_tree (pos, {treedef}, [vmanip])`
```

Comment:
fwiw I'd rather have another api function

Reply:
Would you like it to be part of VoxelManip objects, such that you can use like this: `vm:spawn_tree(pos, treedef)`?
Or would this clutter VoxelManips too much?

Reply:
i suggest keeping it in line with the existing `core.place_schematic_on_vmanip()`